api/zulip/__init__.py

```diff
@@ -219,6 +240,20 @@ def __init__(self, email=None, api_key=None, config_file=None,
             # Default behavior: verify against system CA certificates
             self.tls_verification=True
 
+        if type(client_cert) is type(()):
```

Comment:
I think it might be better to fix this by just passing around two variables anyway, but regardless, this would be most Pythonically written as `if isinstance(client_cert, tuple):` (and is the cause of the Python 3 compatibility test failure).